.env.docker-compose

```
@@ -4,7 +4,7 @@
 ## Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 ## See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="postgresql://postgres:mypassword@localhost:5432/postgres?schema=public"
+DATABASE_URL="postgresql://postgres:mypassword@ttg-postgres:5432/postgres?schema=public"
```

Great fix!

---

Actually I think this should be just `postgres` because Docker uses the service name (`postgres`) for DNS and not the container name (`ttg-postgres`)
```suggestion
DATABASE_URL="postgresql://postgres:mypassword@postgres:5432/postgres?schema=public"
```